engine/pkg/orm/util.go

```diff
@@ -35,6 +41,86 @@ func IsNotFoundError(err error) bool {
 	return strings.Contains(err.Error(), "ErrMetaEntryNotFound")
 }
 
+// InitAllFrameworkModels will create all framework-related tables in SQL backend
+// NOT thread-safe.
+// TODO: What happen if we upgrade the definition of model when rolling update?
+// TODO: need test: change column definition/add column/drop column?
+func InitAllFrameworkModels(ctx context.Context, cc metaModel.ClientConn) error {
+	if cc == nil {
+		return errors.ErrMetaParamsInvalid.GenWithStackByArgs("input client conn is nil")
+	}
+
+	conn, err := cc.GetConn()
+	if err != nil {
+		return err
+	}
+
+	var gormDB *gorm.DB
+	// check if a sql.DB
+	db, ok := conn.(*sql.DB)
+	if ok {
+		gormDB, err = NewGormDB(db)
+		if err != nil {
+			return err
+		}
+	}
+
+	// check if a gorm.DB
+	if gormDB == nil {
+		gormDB, ok = conn.(*gorm.DB)
+		if !ok {
+			return errors.ErrMetaParamsInvalid.GenWithStackByArgs("client conn is not sql DB")
+		}
+	}
```

Comment:
Can extract a function, and reuse it in `InitEpochModel`

Reply:
Fix it.